index.js

```diff
@@ -0,0 +1,61 @@
+// Retrieve data from local storage
+let books = JSON.parse(localStorage.getItem('books')) || [];
+
+// Get HTML elements
+const bookList = document.getElementById('book-list');
+const addBookForm = document.getElementById('add-book-form');
+
+// Function to remove a book from the collection
+function removeBook(index) {
+  books = books.filter((books, i) => i === index);
+  localStorage.setItem('books', JSON.stringify(books));
+}
```

Comment:
- [ ] Kindly make sure to fix the removing method because instead of removing one book, it is removing all books. 

- [OPTIONAL] I highly recommend you filter by using the unique `id` of the book being removed which is a very efficient way to remove the book in your array. Please fix this issue.

  ### APP BEHAVIOUR

   ![chrome-capture-2023-3-24](https://user-images.githubusercontent.com/87197412/234021064-fc596d99-b369-4b99-854f-1b422c771be0.gif)